_posts/en/newsletters/2025-05-23-newsletter.md

```diff
@@ -0,0 +1,166 @@
+
+- **Transaction batching using payjoin:**
+  [Private Pond][private pond post] is an [experimental implementation][private
+  pond github] of a [transaction batching][topic payment batching] service that
+  uses payjoin to generate smaller transactions that pay less in fees.
+
+- **JoinMarket Fidelity Bond Simulator:**
+  The [JoinMarket Fidelity Bond Simulator][jmfbs github] provides tools for
+  JoinMarket participants to simulate their performance in the market based on
+  [fidelity bonds][news161 fb].
+
+- **Bitcoin opcodes documented:**
+  The [Opcode Explained][opcode explained website] website provides
+  documentation for each Bitcoin script opcode.
+
+- **Bitkey code open sourced:**
+  The Bitkey hardware signing device [announced][bitkey blog] their [source
+  code][bitkey github] is open source for non-commercial uses.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [LND 0.19.0-beta.rc5][] is a release candidate for this popular LN
+  node.  One of the major improvements that could probably use testing
+  is the new RBF-based fee bumping for cooperative closes.
+
+- Core Lightning 25.05rc1 FIXME:harding
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #32423][] removes the deprecation notice for
+  `rpcuser/rpcpassword` and replaces it with a security warning about storing
+  cleartext credentials in the configuration file. This option was originally
+  deprecated when `rpcauth` was introduced in [Bitcoin Core #7044][], which
+  supports multiple RPC users and hashes its cookie. The PR also adds a random
+  16-byte salt to credentials from both methods and hashes them before they’re
+  stored in memory.
+
+- [Bitcoin Core #31444][] extends the `TxGraph` class (see Newsletter
+  [#348][news348 txgraph]) with three new helper functions:
+  `GetMainStagingDiagrams()` returns the divergences of clusters between the
+  main and staged feerate diagrams, `GetBlockBuilder()` iterates through graph
+  chunks (sub-cluster feerate-sorted groupings) from highest to lowest feerate
+  for optimized block construction, and `GetWorstMainChunk()` pinpoints the
+  lowest feerate chunk for eviction decisions. This PR is one of the final
+  building blocks before the full implementation of the [cluster mempool][topic
+  cluster mempool] project.
```

Comment:
"before the full implementation of the cluster mempool project" sounds a bit off.

maybe
"before the implementation of the cluster mempool project is complete" or other